satpy/Combined_colormaps.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<p>If someone is using a modern version of pydecorate, why would they get this error?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>That version hasn't been released yet.</p>